### DISCUSSION TITLE
Can I close a popover by click on content?

### DISCUSSION THREAD

**Discussion by Yves852** [unovue/reka-ui#587] [Q&A] [answered]
I would like to close the popoper witch a click on content (either a button or a link with its own action or just by clicking the `<PopoverContent>`). Is there an exposed method or prop for this? 

I tried to wrap the content inside the `<PopoverTrigger>` or inside a second one but did not worked. 

As a workaround I wrapped the `<PopoverContent>` inside a `<PopoverClose>`, it works but maybe not an intented use.

**Comment by k11q** [comment, marked as answer]
I think wrapping with `<PopoverClose>` is a viable solution, I dont think `<PopoverClose>` has any other side effects except closes the popover. Let me know if you come accross a problem
